app/apollo/resolvers/channel.js

```diff
@@ -830,6 +883,9 @@ const channelResolvers = {
         // Deletes the configuration channel
         await models.Channel.deleteOne({ org_id, uuid });
 
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {channel: {name: channel.name, uuid: channel.uuid}};
```

Comment:
Deletion of a Channel **does** automatically delete its versions as well (see `deleteMany` just a couple lines up).  So this is a case where the code _should_ query all the Versions for the config being deleted and include details on those in the pluginContext as well as the channel